src/components/customizableAside.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Image from 'next/image';
+import styles from './customizableAside.module.css';
+
+function CustomizableAside() {
```

Comment:
@Sheyitofunmi Just like I showed you [HERE](https://github.com/AritDeveloperCircle/findyourpeople/pull/60/files#r1181148234), please make this component ACTUALLY customizable. 🙏🏾

Reply:
i have done that ma, i wa having issue will my branch but i resolved it